cmd/podman/pod_create.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var podCreateDescription = "Creates a new empty pod. The pod ID is then" +
+	" printed to stdout. You can then start it at any time with the" +
+	" podman pod start <pod_id> command. The pod will be created with the" +
+	" initial state 'created'."
+
+var podCreateFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "cgroup-to-ctr",
+		Usage: "Tells containers in this pod to use the cgroup created for the pod",
+	},
+	cli.StringFlag{
+		Name:  "cgroup-parent",
+		Usage: "Optional parent cgroup for the pod",
+	},
+	cli.StringSliceFlag{
+		Name:  "label-file",
+		Usage: "Read in a line delimited file of labels (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "label, l",
+		Usage: "Set metadata on pod (default [])",
+	},
+	cli.StringFlag{
+		Name:  "name, n",
+		Usage: "Assign a name to the container",
+	},
+	cli.StringFlag{
+		Name:  "podidfile",
+		Usage: "Write the pod ID to the file",
+	},
+}
+
+var podCreateCommand = cli.Command{
+	Name:                   "create",
+	Usage:                  "create but do not start a pod",
+	Description:            podCreateDescription,
+	Flags:                  podCreateFlags,
+	Action:                 podCreateCmd,
+	ArgsUsage:              "IMAGE [COMMAND [ARG...]]",
+	SkipArgReorder:         true,
+	UseShortOptionHandling: true,
+}
+
+func podCreateCmd(c *cli.Context) error {
+	var options []libpod.PodCreateOption
+	var err error
+
+	if err = validateFlags(c, createFlags); err != nil {
+		return err
+	}
+
+	runtime, err := libpodruntime.GetRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "error creating libpod runtime")
+	}
+	defer runtime.Shutdown(false)
+
+	if c.String("podidfile") != "" {
+		if _, err = os.Stat(c.String("podidfile")); err == nil {
+			return errors.Errorf("pod id file exists. ensure another pod is not using it or delete %s", c.String("podidfile"))
+		}
+		if err = libpod.WriteFile("", c.String("podidfile")); err != nil {
+			return errors.Wrapf(err, "unable to write pod id file %s", c.String("podidfile"))
+		}
+	}
+	// BEGIN GetPodCreateOptions
+
+	// TODO make sure this is correct usage
+	if c.String("cgroup-parent") != "" {
+		options = append(options, libpod.WithPodCgroupParent(c.String("cgroup-parent")))
+	}
+
+	if c.String("cgroup-to-ctr") != "" {
```

Comment:
For bool flags, we prefer `C.IsSet()` or `C.Bool()`

Reply:
whoops stupid copy-paste bug